src/Components/Home/PastGlimpse/PastGlimpse.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./PastGlimpse.scss";
+
+function PastGlimpse() {
+    return (
+        <div className="container">
```

Comment:
For the classname of the parent element, use `past-glimpse` instead of `container`, as `container` classname will be commonly used elsewhere